### PR TITLE
Adds App Engine hosted demo

### PR DESCRIPTION
**Pull request by vpetersson**
* Minor tweaks to asset-metadata example app to support App Engine
* Adds CI/CD workflow for deploying asset-metadata app to App Engine
* Tweaks readme

### CODE REVIEW

File: .github/workflows/deploy-asset-metadata.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Deploy Asset Metadata to App Engine
+
+on:
```
Comment (korvyashka) [inline .github/workflows/deploy-asset-metadata.yml:4]: Let's add path filter - as we don't really need to re-deploy each time master is pushed, as asset-metadata is a separate app.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

File: asset-metadata/Dockerfile [hunk 2]
```diff
@@ -4,11 +4,11 @@ EXPOSE 5000
```
Comment (korvyashka) [inline asset-metadata/Dockerfile:4]: Let's also change EXPOSE to 8080

File: asset-metadata/app.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+runtime: python39
```
Comment (korvyashka) [inline asset-metadata/app.yaml:2]: Image we use is python3.10 - is it ok to run on different runtime?

**Review by korvyashka — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy-asset-metadata.yml (hunk 1 above)
Comment (vpetersson) [reply .github/workflows/deploy-asset-metadata.yml:4]: Good shout!

File: asset-metadata/app.yaml (hunk 3 above)
Comment (vpetersson) [reply asset-metadata/app.yaml:2]: 3.9 is the latest supported by app engine.

**Review by korvyashka — APPROVED** [review]

**Pull request merged**